### PR TITLE
Ignore the TokenAlreadyScheduled error of the reseal_timer

### PR DESCRIPTION
**Pull request by majecty**
This fixes https://github.com/CodeChain-io/codechain/issues/1069

### CODE REVIEW

File: core/src/client/client.rs [hunk 1]
```diff
@@ -286,18 +286,32 @@ impl ResealTimer for Client {
...
-            reseal_timer
+            match reseal_timer
                 .schedule_once(self.importer.miner.get_options().reseal_max_period, RESEAL_MAX_TIMER_TOKEN)
-                .expect("Reseal max timer set succeeds");
+            {
+                Ok(_) => {}
+                Err(TimerScheduleError::TokenAlreadyScheduled) => {
+                    // Since set_max_timer could be called in multi thread, ignore the TokenAlreadyScheduled error
+                }
+                Err(_) => unreachable!("Reseal max timer set succeeds"),
```
Comment (foriequal0) [inline core/src/client/client.rs]: We can't paraphrase the error message as "Expect ... succeeds, but failed". Should we change it as a negative statement?

File: core/src/client/client.rs [hunk 2]
```diff
@@ -286,18 +286,32 @@ impl ResealTimer for Client {
...
+            match reseal_timer
                 .schedule_once(self.importer.miner.get_options().reseal_min_period, RESEAL_MIN_TIMER_TOKEN)
-                .expect("Reseal min timer set succeeds");
+            {
+                Ok(_) => {}
+                Err(TimerScheduleError::TokenAlreadyScheduled) => {
+                    // Since set_min_timer could be called in multi thread, ignore the TokenAlreadyScheduled error
+                }
+                Err(_) => {
+                    unreachable!("Reseal min timer set succeeds");
```
Comment (foriequal0) [inline core/src/client/client.rs]: Same as above. A negative statement?

File: core/src/client/client.rs (hunk 1 above)
Comment (majecty) [reply core/src/client/client.rs]: How about this?
`unreachable!("Reseal max timer should not fail but failed with {:?}", err)`

Comment (foriequal0) [reply core/src/client/client.rs]: Looks Good

**Comment by majecty** [comment]
@sgkim126 @foriequal0 I changed the error message.

**Pull request merged**